Add guide for remote access to integrated ServicePulse (#8436)
Particular/docs.particular.net: servicecontrol/servicecontrol-instances/integrated-servicepulse-remote-access.md

1. Change the Host Name field.
1. Click Save. ServiceControl Management validates the change and restarts the service.

Or edit `ServiceControl.exe.config` directly and restart the service:

```xml
<appSettings>
  <add key="ServiceControl/HostName" value="*" />
</appSettings>
```

> [!WARNING]
> Before changing the host name, confirm that `ServiceControl/DbPath` is present in `ServiceControl.exe.config`. When that setting is absent, the default database folder name is derived from the host name and port, so changing either one points ServiceControl at a different, empty database. Instances deployed with ServiceControl Management or PowerShell have `ServiceControl/DbPath` written explicitly and are unaffected. See [RavenDB embedded location](/servicecontrol/configure-ravendb-location.md).

Containers already accept connections addressed to any host name and need no change; expose the container's port `33333` instead.

### Choosing a port

Now that users type this address themselves, the default port `33333` is worth reconsidering. It can be changed with [`ServiceControl/Port`](configuration.md#host-settings-servicecontrolport).

> The port worth choosing is `443`, the standard port for HTTPS. Browsers hide it, so users get a clean address with no port number at all:

Reviewer: ```suggestion
The recommended port is `443`, the standard port for HTTPS. Browsers hide it, so users get a clean address with no port number.
```

- any other tools or scripts calling the API directly are updated, along with the [`ServiceControl/RemoteInstances`](configuration.md#host-settings-servicecontrolremoteinstances) configuration of any *other* Error instance configured to read from this one, which only applies to federated setups

Alternatively, leave ServiceControl's port alone and put a [reverse proxy](#using-a-reverse-proxy-instead) in front to publish it on `443`.

## Step 2: Allow the port through the firewall

Add an inbound firewall rule on the ServiceControl machine for the Error instance port (`33333` by default). On Windows this can be done through the Windows Defender Firewall console or with PowerShell:

```powershell
New-NetFirewallRule -DisplayName "ServiceControl (integrated ServicePulse)" -Direction Inbound -Protocol TCP -LocalPort 33333 -Action Allow
```

Microsoft documents both approaches in [Windows Firewall rules](https://learn.microsoft.com/en-us/windows/security/operating-system-security/network-security/windows-firewall/rules) and the [`New-NetFirewallRule` reference](https://learn.microsoft.com/en-us/powershell/module/netsecurity/new-netfirewallrule).

> Network firewalls, proxies, or DNS between users and the machine may also need changes. If any of this is managed by another team, this is the point to involve whoever looks after the network and security in the organization: the request is to allow inbound TCP traffic to this port on this machine, from the users who need access.

Reviewer: ```suggestion
Network firewalls, proxies, or DNS between users and the machine may also need changes. If another team is responsible for the network or security infrastructure, involve them at this point. They should configure the network to allow inbound TCP traffic to this port on this machine from the users who require access.
```

## Step 4: Make ServiceControl Monitoring reachable

> [!NOTE]
> Skip this step if [monitoring](/monitoring/) is not in use. To hide the Monitoring tab, set the monitoring connection to `!` as described in [Configuring the monitoring URL](#step-4-make-servicecontrol-monitoring-reachable-configuring-the-monitoring-url).

Monitoring data comes from a separate [ServiceControl Monitoring instance](/servicecontrol/monitoring-instances/) with its own host name and port. Unlike the Error instance API, it does not share an address with ServicePulse, so the browser has to be told where to find it.

The default is `http://localhost:33633/`, and in a browser on someone else's machine `localhost` means *their* machine, not the ServiceControl machine. The Monitoring tab therefore fails to load until the following are configured.

1. Set [`Monitoring/HttpHostname`](/servicecontrol/monitoring-instances/configuration.md#host-settings-monitoringhttphostname) on the Monitoring instance so it accepts connections from other machines, and allow its port (`33633` by default) through the firewall as in [Step 2](#step-2-allow-the-port-through-the-firewall).

> 1. Allow the ServicePulse address on the Monitoring instance with [`Monitoring/Cors.AllowedOrigins`](/servicecontrol/monitoring-instances/configuration.md#cors-monitoringcors-allowedorigins). Because the Monitoring instance is at a different address from ServicePulse, browsers require it to opt in to these requests, even though this is not needed for the Error instance API:

Reviewer 2: ```suggestion
1. Decide which origins the Monitoring instance accepts requests from. Because it is at a different address from ServicePulse, its [CORS](/servicecontrol/security/configuration/cors.md) policy governs these requests, unlike calls to the Error instance API. [`Monitoring/Cors.AllowAnyOrigin`](/servicecontrol/monitoring-instances/configuration.md#cors-monitoringcors-allowanyorigin) defaults to `true`, so the Monitoring tab works without configuring anything here. For production, set it to `false` and list the ServicePulse address in [`Monitoring/Cors.AllowedOrigins`](/servicecontrol/monitoring-instances/configuration.md#cors-monitoringcors-allowedorigins) instead:
```
